### DISCUSSION TITLE
How do I interpret the Date axis along the bottom of the AstroCalc>PC>Graphs tab?

### DISCUSSION THREAD

**Discussion by adgorn** [Stellarium/stellarium#2985] [Q&A]
The Guide says it represents "a range of 600 days (centered on the current date)".  I'm not getting how that shows on the graph.  Thanks.

**Comment by alex-w** [comment]
Current date in center of graph, whole graph has a range of 600 days.

**Reply by adgorn** [reply]
I see now that I can expand the default size of the window to see more of the axis details.  And also that you are using the convention of day.month for the date format, which is backwards from the way we do it here.  Thanks for your help.
